website/content/docs/connect/registration/service-registration.mdx

```diff
@@ -117,6 +118,10 @@ registering a proxy instance.
   Defaults to the port advertised by the service instance identified by
   `destination_service_id` if it exists otherwise it may be empty in responses.
 
+- `local_service_socket_path` - The path of a unix domain socket to connect to the local application
```

Comment:
Both `unix` and `Unix` are used — do we have a stylistic guide which recommends one over the other, for consistency?